src/generate/device.rs

```diff
@@ -112,7 +112,7 @@ pub fn render(
         out.extend(quote! {
             pub use cortex_m::peripheral::Peripherals as CorePeripherals;
             #[cfg(feature = "rt")]
-            pub use cortex_m_rt::interrupt;
```

Comment:
I think this was here to ensure the svd2rust generated PAC exports an `interrupt` macro, since the cortex-m-rt documentation says you should use `interrupt` from your PAC and not from c-m-rt. So, I think we should keep the `pub use cortex_m_rt::interrupt`, but we don't need to `pub use cortex_m::interrupt::InterruptNumber` at all, since the impl writes the full path anyway.